buildozer/targets/ios.py

```diff
@@ -3,9 +3,6 @@
 '''
 
 import sys
-if sys.platform != 'darwin':
-    raise NotImplementedError('Windows platform not yet working for Android')
```

Comment:
We got the message a bit wrong, but also this got moved to `check_requirements()` instead. It feels more appropriated and improves testability